io.github.neph_io.dsost.yml

```
@@ -0,0 +1,37 @@
+app-id: io.github.neph_io.dsost
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-23.08'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+
+command: /app/bin/launch.sh
```

```suggestion
command: launch.sh
```

this should never be an absolute path.